holocron_api/api/models/standard.py

```diff
@@ -21,23 +35,41 @@ def __str__(self):
         return self.campaign_key
 
 
+class Tactic(models.Model):
+    tactic_id = models.AutoField(primary_key=True)
+    tactic_key = models.CharField(max_length=100, blank=True, null=True,
```

Comment:
If we're really making it open source I can see people wanting to use a key there for the URL instead of the id.